src/ControlTasks/display_frame.py

```diff
@@ -0,0 +1,20 @@
+# break up into smaller control tasks as needed
```

Comment:
Make sure you have documentation on all your classes- there's a few in a row here without any